tools/pipelines/templates/build-npm-package.yml

```diff
@@ -620,3 +628,38 @@ stages:
         namespace: ${{ parameters.namespace }}
         tagName: ${{ parameters.tagName }}
         publishNonScopedPackages: ${{ variables.publishNonScopedPackages }}
+
+  # Capture per-pipeline stage results
+  - stage: runAfterAll
+    displayName: Retrieve and upload client stage stats to Kusto
+    condition: and(succeededOrFailed(), ne(variables['Build.Reason'], 'PullRequest'), eq(${{parameters.telemetry}}, true))
```

Comment:
Before checking in, I'll change the Build.Reason condition to check for IndividualCI.

Reply:
Yep, forgot to mention that!